### PR TITLE
Revise function name in /export/__init__.py

### PR DESCRIPTION
**Pull request by CBroz1**
This was missed in a previous PR wherein a code review suggested the updated name. 

Also added updated diagram

### CODE REVIEW

**Comment by CBroz1** [comment]
@kabilar - I think this would be relatively quick review if you have time for it

File: element_lab/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# from . import lab
```
Comment (kabilar) [inline element_lab/__init__.py:1]:
```suggestion
```

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!  Only minor feedback would be if you could please remove the schema name from the diagram.

**Comment by CBroz1** [comment]
> Thanks @CBroz1! Only minor feedback would be if you could please remove the schema name from the diagram.

Thanks for taking a look @kabilar, all set

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!

**Pull request merged**